tests/unit/test-class-wc-payments-order-service.php

```diff
@@ -1206,13 +1206,9 @@ public function test_get_fraud_meta_box_type() {
 	}
 
 	public function test_attach_intent_info_to_order() {
-		$intent_id      = 'pi_mock';
-		$intent_status  = 'succeeded';
-		$payment_method = 'woocommerce_payments';
-		$customer_id    = 'cus_12345';
-		$charge_id      = 'ch_mock';
-		$currency       = 'USD';
-		$this->order_service->attach_intent_info_to_order( $this->order, $intent_id, $intent_status, $payment_method, $customer_id, $charge_id, $currency );
+		$intent_id = 'pi_mock';
+		$intent    = WC_Helper_Intention::create_intention( [ 'id' => $intent_id ] );
```

Comment:
since `pi_mock` is used only in once place, it makes no sense to define a variable for it

Reply:
Good point. Fixed in https://github.com/Automattic/woocommerce-payments/pull/7945/commits/1f4681ac38ffcf9e6eca44eef15927205cc5a9d4.

Reply:
Whoops, the variable is being used here too:  
https://github.com/Automattic/woocommerce-payments/blob/1f4681ac38ffcf9e6eca44eef15927205cc5a9d4/tests/unit/test-class-wc-payments-order-service.php#L1218

Reverting the change.